templates/visuals/.api/v1.7.0/schema.capabilities.json

```diff
@@ -686,6 +686,10 @@
                 "type": {
                     "description": "Describes what type of property this is and how it should be displayed to the user",
                     "$ref": "#/definitions/valueType"
+                },
+                "rule": {
+                    "type": "object",
+                    "description": "Describes some custom logic rule. For instance, it uses for gradient option substitution"
```

Comment:
what is this needed for? not referenced...

Reply:
Ok. I will provide exact description.